### DISCUSSION TITLE
Add support admin hub localization

### DISCUSSION THREAD

**Discussion by devmansurov** [lunarphp/lunar#55] [Feature Requests]
Add pleace support admin hub localization

**Comment by glennjacobs** [comment]
At the moment the hub does use language files, but of course they're only in English ("en"). As we do not speak other languages (which is a shame, I'd like to!), we are unable to support other languages in the core. 

So we suggest the community create packages to register other language files. 

As for switching languages, this is something we'll look to add in the coming months on staff profiles.

**Reply by devmansurov** [reply]
I can help with translations in Russian, Tajik and Uzbek langs

**Reply by yaser2490** [reply]
I can help with Persian translation

Some languages, such as Persian, require the page to be RTL
I hope this feature is added to the panel as soon as possible

The ability to change the admin font is also a very good feature (Persian language needs this feature)

**Reply by glennjacobs** [reply]
Thanks. We'll be looking at this soon. As we're still changing the UI whilst in beta, we don't want to start translations too early.

**Reply by timmydhooghe** [reply]
I can help with Dutch translations once you're ready. 🙂

@glennjacobs I'll subscribe to this discussion. Are you planning to update us here once you're ready to accept language contributions?

**Reply by glennjacobs** [reply]
I am happy to receive PRs for language translations.

**Reply by timmydhooghe** [reply]
Awesome! 
I see some PR's to main, and some to 0.6, what's the preferred way?

**Reply by glennjacobs** [reply]
I would suggest `main`

**Comment by rinodrummer** [comment]
Hi guys! Me and @sebhd1 made a draft PR for Italian translations in #1290!

Right now I'm merging on `0.6` because I synched my branch with it before reading @glennjacobs's comment on @timmydhooghe's question about the preferred branch.

As I said in my PR:

> We translated all the entries in the localization files, but noticed that most of the UI components have un-wrapped texts that make them hard to translate.
I think that a guide line must be discussed to define which namespaces must be attributed to some components and must enforce to use __() whenever possible, even in component labels, placeholders and so on when new hub section are introduced.

**Reply by timmydhooghe** [reply]
Awesome, I made a draft PR for Dutch translations, #1283. Haven't had the chance to check the copy in the frontend yet though.
